### ISSUE TITLE
Missing tiles for 11 participiants

### ISSUE THREAD

**Issue by akiziltoprak** [ant-media/conference-call-application#187]
Although there are 11 participants, tiles only left at 9

**Comment by lastpeony** [comment]
After thorough investigation, I have uncovered the following insights:

The primary cause of the issue lies in a bug within the VIDEO_TRACK_ASSIGNMENT_LIST callback, which has been addressed and resolved in the following PR:
https://github.com/ant-media/conference-call-application/pull/231
See the changes on line 1308
https://github.com/ant-media/conference-call-application/pull/231/files#diff-c787034ab83567f7cc7704897b0e06f4c31efe493b09bfc00a2c735175880a89R1308


![Image](https://github.com/ant-media/conference-call-application/assets/24750845/1aba82a9-c591-4c13-9532-925b13073c7c)


Additionally, it's essential to note the following point for consideration:
If the maxVideoTrackCount is set to 6 (default) in the application settings, and an 8th participant joins the room, even though the tile count on the client side is capped at 30, their tile won't be displayed on the screen.

This discrepancy occurs because the tiles on the screen are rendered using the setParticipants() setState function. This function is invoked when a "newStreamAvailable" message is received from the server. However, if the maxVideoTrack count is reached on the server side, the "newStreamAvailable" message isn't sent by the server. Consequently, the tile for the newly joined participant isn't drawn on the client's screen when the participant count exceeds 7 (upon the 8th join).

To address this issue, use the provided PR, and set the maxVideoTrackCount to 30 in the application settings. Upon testing, you will observe that the server sends "newStreamAvailable" messages for the 8th, 9th, 10th, and subsequent participants. This ensures that all new joiner tiles will be rendered on the screen.
